src/app/pages/virtrolio-cover/virtrolio-cover.component.html

```diff
@@ -18,43 +18,59 @@ <h2 id="users-virtrolio">User's virtrolio</h2>
     <!-- Side buttons -->
     <div class="col-md-2 col-sm-3 px-2 mt-4 mt-md-0 vertical-center">
       <div id="buttons">
-        <div class="row">
-          <button class="btn-lg btn-primary" type="button">Export to PDF</button>
-        </div>
-        <div class="row">
-          <button class="btn-lg btn-primary" type="button" data-toggle="modal" data-target="#link-gen">
-            Sharing Link
-          </button>
-        </div>
-        <div class="row">
-          <button class="btn-lg btn-primary" type="button" routerLink="/settings">Settings</button>
-        </div>
+        <div class="row"><button class="btn-lg btn-primary" type="button" data-toggle="modal"
```

Comment:
You should not start a div and a button on the same line.
```suggestion
        <div class="row">
          <button class="btn-lg btn-primary" type="button" data-toggle="modal"
```